### PR TITLE
feat: table view switch to the graph component

### PR DESCRIPTION
**Pull request by sagarnaikjuspay**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description
add a table view to the line graph component 
image

## How did you test it?
enable the new analytics feature flag and the line graph module the table tab should show the json dumb on the place of line graph

## Where to test it?

- [x] INTEG
- [x] SANDBOX
- [x] PROD

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

**Review by JeevaRamu0104 — DISMISSED** [review]

File: src/screens/NewAnalytics/OverViewAnalytics/PaymentsProcessed/PaymentsProcessed.res [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  }
+  React.useEffect(() => {
+    getPaymentsProcessed()->ignore
+    None
+  }, [])
+
+  <div>
+    <ModuleHeader title={entity.title} />
+    <Card>
+      <GraphHeader title="165K USD" viewType setViewType />
```
Comment (gitanjli525) [inline src/screens/NewAnalytics/OverViewAnalytics/PaymentsProcessed/PaymentsProcessed.res:75]: is this title hard coded intentionally ?

Comment (sagarnaikjuspay) [reply src/screens/NewAnalytics/OverViewAnalytics/PaymentsProcessed/PaymentsProcessed.res:75]: yeah will be replaced by dynamic value once the api's are ready

File: src/screens/NewAnalytics/OverViewAnalytics/PaymentsProcessed/PaymentsProcessed.res [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  let getPaymentsProcessed = async () => {
+    try {
+      let data =
+        getData
+        ->LogicUtils.getDictFromJsonObject
+        ->LogicUtils.getArrayFromDict("queryData", [])
+
+      setpaymentsProcessed(_ => data)
+    } catch {
+    | _ => ()
```
[the remark this answers is not in the document]
Comment (sagarnaikjuspay) [reply src/screens/NewAnalytics/OverViewAnalytics/PaymentsProcessed/PaymentsProcessed.res:64]: `let getPaymentsProcessed = async () => {
    try {
      let data =
        getData
        ->LogicUtils.getDictFromJsonObject
        ->LogicUtils.getArrayFromDict("queryData", [])

      setpaymentsProcessed(_ => data)
    } catch {
    | _ => ()
    }
  }`
  
  the api data is hardcoded currencltly so no error will be thrown, will add a page loaded snippets once the api's are ready

File: src/screens/NewAnalytics/OverViewAnalytics/OverViewAnalyticsUtils.res [hunk 3]
```diff
@@ -1,26 +1 @@
...
-    name: "Series 1",
-    data: [3000, 5000, 7000, 5360, 4500, 6800, 5400, 3000, 0, 0],
-    color: "#2f7ed8",
-  }
-  let title = {
-    text: "USD",
-  }
-  {categories, data, title}
-}
+
```
[the remark this answers is not in the document]
Comment (sagarnaikjuspay) [reply src/screens/NewAnalytics/OverViewAnalytics/OverViewAnalyticsUtils.res:1]: if there's are common functions btw modules will keep this file, or else remove later

**Review by gitanjli525 — APPROVED** [review]

File: src/screens/NewAnalytics/Graphs/LineGraph/LineGraph.res [hunk 4]
```diff
@@ -1,8 +1,18 @@
...
+let make = (~entity, ~data: JSON.t, ~className="") => {
   open NewAnalyticsTypes
   let data = entity.getObjects(data)
-  let options = entity.getChatOptions(data)
-  <Highcharts.Chart options={options->lineGraphOptionsToJson} highcharts={Highcharts.highcharts} />
+  let default = entity.getChatOptions(data)->lineGraphOptionsToJson
+  let (options, setOptions) = React.useState(_ => default)
+
+  React.useEffect(() => {
+    // to re-rendor the chart
```
Comment (PritishBudhiraja) [inline src/screens/NewAnalytics/Graphs/LineGraph/LineGraph.res:10]: why this comment self explanatory.

**Review by PritishBudhiraja — APPROVED** [review]

**Review by Riddhiagrawal001 — APPROVED** [review]

**Pull request merged**